File: 2021/vika3/msp.cpp
#include<algorithm>
int m(int*a,int*b,int n){
 std::sort(a,a+n);
 std::sort(b,b+n);
 int r=0;
 while(--n>=0)
  r+=a[n]*(*b++);
return r;
}

int main(){
    int* a = {1,3,-5};
    int* b = {-2,4,1};
}

File: 2021/vika03/msp.py
def msp(a, b):
    a.sort()
    a.reverse()
    b.sort()
    summa = 0
    for i in range(len(a)):
        summa += a[i] * b[i]
    return summa


T = int(input())
for i in range(T):
    n = input()
    A = input()
    a = list(map(int,A.split()))
    B = input()
    b = list(map(int,B.split()))
    print("Case #"+str(i+1)+":", msp(a,b))
